tests/queries/0_stateless/03237_array_auc_unscaled.sql

```diff
@@ -0,0 +1,30 @@
+select arrayAUC([0.1, 0.4, 0.35, 0.8], [0, 0, 1, 1], false);
```

Comment:
People have different opinions on adding new test files vs. extending existing tests with new test cases. I am in the latter camp as related tests are kept together.

Could you move the new tests into 01064_array_auc.sql (+ adjust its .reference file). Please also check that the test is as systematic as possible so we don't miss edge cases. Thanks.

Reply:
done, also added tests to validate passing True works as before and that the new exceptions are properly raised.